CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 * **BREAKING:** Drop libsass support ([PR #4106](https://github.com/alphagov/govuk_publishing_components/pull/4106))
 * **BREAKING:** Drop support for legacy browsers ([PR #4111](https://github.com/alphagov/govuk_publishing_components/pull/4111))
 
+## Unreleased
+
+* Add 'Chat entry' component ([PR #4151](https://github.com/alphagov/govuk_publishing_components/pull/4151))
```

Comment:
Something weird with the CHANGELOG here - this needs to be above `42.0.0`.

Reply:
Thanks, fixed now.